api/src/main/resources/Reference_Application_Concepts-21.xml

```diff
@@ -15,106 +15,106 @@
   <concept_class concept_class_id="12" name="Symptom" description="Patient-reported observation" creator="1" date_created="2004-10-04 00:00:00.0" retired="false" retired_by="1" uuid="8d492954-c2cc-11de-8d13-0010c6dffd0f"/>
   <concept_class concept_class_id="13" name="Symptom/Finding" description="Observation that can be reported from patient or found on exam" creator="1" date_created="2004-10-04 00:00:00.0" retired="false" retired_by="1" uuid="8d492b2a-c2cc-11de-8d13-0010c6dffd0f"/>
   <concept_class concept_class_id="24" name="Pharmacologic Drug Class" description="Class of medications based on pharmacologic properties as opposed to therapeutic properties" creator="1" date_created="2014-02-07 22:16:02.0" retired="false" retired_by="1" uuid="b4535251-9183-4175-959e-9ee67dc71e78"/>
-  <concept concept_id="108" retired="false" datatype_id="4" class_id="12" is_set="false" creator="1" date_created="2004-01-01 00:00:00.0" version="" changed_by="1" date_changed="2015-11-05 01:34:22.0" retired_by="1" uuid="108AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="121" retired="false" datatype_id="4" class_id="5" is_set="false" creator="1" date_created="2004-01-01 00:00:00.0" version="" changed_by="1" date_changed="2015-11-05 01:37:39.0" retired_by="1" uuid="121AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="512" retired="false" datatype_id="2" class_id="13" is_set="false" creator="1" date_created="2004-01-01 00:00:00.0" version="596285" changed_by="1" date_changed="2016-04-27 14:18:52.0" retired_by="1" uuid="512AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="924" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2004-07-23 00:00:00.0" version="" changed_by="1" date_changed="2010-10-01 14:33:22.0" retired_by="1" uuid="924AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="1065" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2005-01-06 00:00:00.0" version="" changed_by="1" date_changed="2016-08-16 07:47:47.0" retired_by="1" uuid="1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="1066" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2005-01-06 00:00:00.0" version="" changed_by="1" date_changed="2016-08-16 07:48:25.0" retired_by="1" uuid="1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="1067" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2005-01-06 00:00:00.0" version="" changed_by="1" date_changed="2016-08-16 07:46:41.0" retired_by="1" uuid="1067AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="1284" retired="false" datatype_id="2" class_id="5" is_set="false" creator="1" date_created="2005-02-24 12:33:10.0" version="" changed_by="1" date_changed="2016-07-10 02:53:00.0" retired_by="1" uuid="1284AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```

Comment:
Did you move around all these many lines? I thought i would see only a one line change.

Reply:
yes. I changed all the lines.

Reply:
yes. I changed all the lines. Isn't that what was needed?